### PR TITLE
Added VDE Import and Lang contracts

### PR DESCRIPTION
**Pull request by gentee**

### CODE REVIEW

**Review by potehinre — APPROVED** [review]

File: packages/api/vde_test.go [hunk 1]
```diff
@@ -316,6 +316,34 @@ func TestVDEParams(t *testing.T) {
...
+	value := `{"en": "My VDE test", "fr": "French VDE test"}`
+
+	form = url.Values{"Name": {name}, "Trans": {value}, "vde": {`true`}}
+	err = postTx(`NewLang`, &form)
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	input := fmt.Sprintf(`Span($%s$)+LangRes(%[1]s,fr)`, name)
+	var lret contentResult
```
Comment (ghost) [inline packages/api/vde_test.go]: let's use more semantic variable name.

File: packages/smart/smart.go [hunk 2]
```diff
@@ -85,7 +85,11 @@ func GetTestValue(name string) string {
 
 // GetLogger is returning logger
 func (sc SmartContract) GetLogger() *log.Entry {
-	return log.WithFields(log.Fields{"vde": sc.VDE, "name": sc.TxContract.Name})
+	var name string
```
Comment (ghost) [inline packages/smart/smart.go]: Useless comment

File: packages/smart/smart_p.go [hunk 3]
```diff
@@ -498,7 +498,7 @@ func HexToBytes(hexdata string) ([]byte, error) {
 
 // LangRes returns the language resource
 func LangRes(sc *SmartContract, idRes, lang string) string {
-	ret, _ := language.LangText(idRes, int(sc.TxSmart.EcosystemID), lang)
+	ret, _ := language.LangText(idRes, int(sc.TxSmart.EcosystemID), lang, sc.VDE)
```
Comment (ghost) [inline packages/smart/smart_p.go]: useless comment

File: packages/smart/smart_p.go [hunk 4]
```diff
@@ -681,7 +681,7 @@ func RollbackColumn(sc *SmartContract, tableName, name string) error {
 
 // UpdateLang updates language resource
 func UpdateLang(sc *SmartContract, name, trans string) {
-	language.UpdateLang(int(sc.TxSmart.EcosystemID), name, trans)
+	language.UpdateLang(int(sc.TxSmart.EcosystemID), name, trans, sc.VDE)
```
Comment (ghost) [inline packages/smart/smart_p.go]: useless comment

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**